### PR TITLE
[AFF] Get data for serialization from the DB

### PR DESCRIPTION
**Pull request by albertmink**
This is definitely not 7.02 syntax, however it is function working in our dev system.

Next steps: ⏩ downport syntax to 7.02 and make abaplint ✔️  

Contributes to #5658

### CODE REVIEW

File: src/objects/zcl_abapgit_object_intf.clas.locals_imp.abap [hunk 1]
```diff
@@ -1,6 +1,419 @@
...
+
+    CALL FUNCTION 'SEO_BUFFER_REFRESH'
+      EXPORTING
+        cifkey  = lo_cifkey
+        version = seoc_version_inactive.
+
+    CALL FUNCTION 'SEO_CLASS_GENERATE_CLASSPOOL'
+      EXPORTING
+        clskey        = lo_cifkey
+        suppress_corr = seox_true
```
Comment (larshp) [inline src/objects/zcl_abapgit_object_intf.clas.locals_imp.abap:134]: add `seox_true` to constants in linter config

**Comment by albertmink** [comment]
Need help to fix the GitHub action findings/errors @larshp

Seams as if the ignored constants (`seoo_cmptype_method` and others) are not ignored in the standalone report.

**Comment by larshp** [comment]
also add the constants to https://github.com/abapGit/abapGit/blob/main/ci/abaplint.json

**Comment by albertmink** [comment]
merge?

**Pull request merged**